website/content/en/docs/kubebuilder/quickstart.md

```diff
@@ -260,6 +260,103 @@ The `ClusterRole` manifest at `config/rbac/role.yaml` is generated from the abov
 $ make manifests
 ```
 
+### Create a validating or mutating Admission Webhook (optional)
+
+An admission webhook is an HTTP callback that is registered with Kubernetes, and will be called by Kubernetes to validate or mutate
+a resource before being stored. There are two types of admission webhooks, validating and mutating. Validating webhooks can be used
```

Comment:
why we break the lines, I mean should note that line 266 `a resource ...` be in the above line after mutate for example? Is not possible that it break the render of the text with we change the css?

Reply:
It shouldn't affect the rendering at all as far as I understand, the intent was to make it more readable in the diff but I was obviously a few characters off

Reply:
IMO all lines should be 80 characters long :)